src/main/java/core/basesyntax/Application.java

```diff
@@ -1,7 +1,15 @@
 package core.basesyntax;
 
+import java.util.Arrays;
+
 public class Application {
     public static void main(String[] args) {
         // create three balls using class Lottery and print information about them in console
+        Ball[] balls = new Ball[3];
+        for (int i = 0; i < balls.length; i++) {
+            Ball ball = new Lottery().getRandomBall();
+            balls[i] = ball;
+        }
+        System.out.println(Arrays.toString(balls));
```

Comment:
Pay attention to access modifiers. The `Application` class should have an access modifier to indicate its visibility explicitly, even if it's public.